acp/README.md

```diff
@@ -436,8 +436,14 @@ To perform authenticated operations you will need to build and sign a JWT token
 
 - `sub` public key of the identity
 - `aud` host name of the defradb api
-
-> The `exp` and `nbf` fields should also be set to short-lived durations.
+- The `exp` and `nbf` fields should also be set to short-lived durations.
```

Comment:
suggestion: Perhaps a better separation of local and sourcehub acp specific options as they might use them interchangeably or be confusing if an option is needed for xyz acp type.

Reply:
There are no local only fields. And the sourcehub only ones are very clearly documented as such

Reply:
Maybe a sourcehub acp dedicated section somewhere in this markdown file guiding users how to setup sourcehub acp and all the bits they would need, and how to get them?

Reply:
I do think that should be covered by SourceHub documentation.  We have to bear in mind that SourceHub is a separate service/process too, and it's version is not tied to Defra releases.  By duplicating documentation in the Defra repo it will likely be incorrect at various times as SourceHub updates.